from dung.battle_exceptions.battle_exception import BattleException


class HeroEvadedBattleException(BattleException):
    pass